README.md

```diff
@@ -49,8 +49,17 @@ yarprobotinterface --config conf/launch_wholebodydynamics_ecub.xml
 ```
 
 ## URDF generation
-This repository hosts the configuration files for generating ergoCub urdf via [`creo2urdf`](https://github.com/icub-tech-iit/creo2urdf).
-You can find there the relative documentation on how write those configuration files.
+
+This repository hosts the configuration files for generating ergoCub urdf. To generate the URDF, you need to have access to the `cad-mechanics` repo (that is currently private, if you need access ask it to the ergocub-software mantainer) and install the following repos and software:
+* You need to install the version of Creo required by [`cad-mechanics`](https://github.com/icub-tech-iit/cad-mechanics/).
```

Comment:
I've noticed that https://github.com/icub-tech-iit/cad-mechanics/ is private, is it wanted? I've also noticed that I don't have the access  😃

Reply:
In theory that is documented, see the sentence "(that is currently private, if you need access ask it to the ergocub-software mantainer)".

Reply:
I guess it could make sense to consider moving those models to `cad-mechanics-public`, but I did not want to block the docs improvement on that discussion.

Reply:
> I guess it could make sense to consider moving those models to cad-mechanics-public

cc @fiorisi @maggia80